src/modules/settings/components/settings/twitch/AnonChat.jsx

```diff
@@ -22,17 +23,48 @@ function AnonChat() {
         </p>
         <Toggle checked={value} onChange={(state) => setValue(state)} />
       </div>
-      <div className={styles.settingRow}>
-        <p className={styles.settingDescription}>
-          {formatMessage({defaultMessage: 'Whitelist channels that bypass Anon Chat'})}
-        </p>
-        <TagInput
-          value={channels}
-          className={styles.settingTagInput}
-          onChange={(newValue) => setChannels(newValue)}
-          placeholder={formatMessage({defaultMessage: 'username, etc..'})}
-        />
-      </div>
+      {value ? (
+        <>
+          <div className={styles.settingRow}>
+            <p className={styles.settingDescription}>
+              {filter === AnonChatFilterTypes.WHITELIST
+                ? formatMessage({defaultMessage: 'Whitelist channels that bypass Anon Chat'})
+                : formatMessage({defaultMessage: 'Blacklist channels that enable Anon Chat'})}
+            </p>
+            <TagInput
+              value={channels}
+              className={styles.settingTagInput}
+              onChange={(newValue) => setChannels(newValue)}
+              placeholder={formatMessage({defaultMessage: 'username, etc..'})}
+            />
+          </div>
+          <div className={styles.settingRowRightAlign}>
+            {formatMessage(
+              {defaultMessage: 'Use <button>{filterType}</button> instead.'},
+              {
+                filterType: filter === AnonChatFilterTypes.WHITELIST ? 'Blacklist' : 'Whitelist',
+                // eslint-disable-next-line react/no-unstable-nested-components
+                button: (text) => (
+                  <Button
```

Comment:
it might be a bit nicer to swap this to be two separate settings.. would allow you to set channels in either list